.github/workflows/build.yaml

```diff
@@ -41,6 +41,9 @@ jobs:
       - name: Build (default features)
         run: cargo rustc --locked --bin cloud-hypervisor -- -D warnings
 
+      - name: Build (default features + guest_debug)
+        run: cargo rustc --locked --bin cloud-hypervisor --features "guest_debug" -- -D warnings
+
```

Comment:
Looks good, but could you please move this after the `gdb` one?

Reply:
Okay.